deployer/src/deployment/queue.rs

```diff
@@ -379,6 +379,7 @@ async fn run_pre_deploy_tests(
 
     let stdout = cmd.stdout.take().unwrap();
     let stdout_reader = BufReader::new(stdout);
+    #[allow(clippy::lines_filter_map_ok)]
```

Comment:
Yeah, I think I could refactor it to be like the cargo PR. I just put a lint ignore because the suggested solution just triggered several more clippy lints.

Reply:
Sounds good! I see we don't handle stderr here like you do in your PR, but I don't think we need to fix that in this PR.